app/components/atomic/galoy-primary-button/galoy-primary-button.tsx

```diff
@@ -26,7 +26,7 @@ export const GaloyPrimaryButton: FunctionComponent<PropsWithChildren<ButtonProps
 
 const useStyles = makeStyles((theme) => ({
   titleStyle: {
-    fontSize: 20,
+    fontSize: 28,
```

Comment:
Is this just a preference change?

Reply:
> Is this just a preference change?

the text is too small otherwise (imo, at least)

Reply:
Thats fair. In that case you should adjust the line height to be `32` or something bigger than the font size.

Reply:
updated